modules/instrument_list/php/instrument_list_controlpanel.class.inc

```diff
@@ -280,27 +280,25 @@ class Instrument_List_ControlPanel extends \TimePoint
 
         // set the BVLQC flag
         $user =& \User::singleton();
-        if (isset($_REQUEST['setBVLQCStatus'])
+        if (isset($_REQUEST['setBVLQCStatus']) && !empty($_REQUEST['setBVLQCStatus'])
```

Comment:
I believe `empty()` already checks for whether a variable exists so `isset` here is not neccesary?